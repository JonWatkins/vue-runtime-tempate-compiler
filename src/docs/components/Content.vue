<template>
  <main
    class="col-md-9 col-xl-9 offset-md-3 offset-xl-3 py-md-3 px-md-5 content"
  >
    <router-view></router-view>
  </main>
</template>

<script>
export default {
  name: 'Content'
}
</script>
