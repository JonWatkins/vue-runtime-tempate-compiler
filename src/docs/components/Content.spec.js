import { shallowMount } from '@vue/test-utils'
import Content from './Content.vue'
const { localVueInst } = global

let wrapper

describe('Content', () => {
  beforeEach(() => {
    wrapper = shallowMount(Content, {
      localVue: localVueInst
    })
  })

  it('is called', () => {
    expect(wrapper.name()).toBe('Content')
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it('render correctly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
