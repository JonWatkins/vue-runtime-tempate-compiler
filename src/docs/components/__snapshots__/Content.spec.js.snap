// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content render correctly 1`] = `
<main class="col-md-9 col-xl-9 offset-md-3 offset-xl-3 py-md-3 px-md-5 content">
  <router-view-stub name="default"></router-view-stub>
</main>
`;
